packages/blob/src/index.ts

```diff
@@ -215,8 +211,60 @@ export async function list(
   };
 }
 
-function getApiUrl(): string {
-  return process.env.VERCEL_BLOB_API_URL || 'https://blob.vercel-storage.com';
+export interface GenerateClientTokenOptions extends BlobCommandOptions {
+  pathname: string;
+  onUploadCompletedUrl: string;
+  onUploadCompletedCallbackUrlArgs?: Record<string, string>;
+}
+
+export function generateClientTokenFromReadWriteToken({
```

Comment:
@vvo despite the additional network request, I think moving this function to the vercel api (or the worker eventually) would allow us to update the client token format without breaking changes on the `npm` module. wdyt?

Reply:
> would allow us to update the client token format without breaking changes on the npm module.

What kind of changes can you foresee at the moment? The fact that even AWS did it this way makes me think this is a correct way, even if you ever have to implement changes.